common/changes/@bentley/ui-components/presentation-highlighted-matches_2020-11-18-11-58.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@bentley/ui-components",
+      "comment": "updated [[FilteringInput]]component",
```

Comment:
This message is added to a changelog that's supposed to be read by package consumers. The message has to be short but readable and detailed enough to understand what changes were made that affect consumers.

Reply:
Changed the message

Reply:
I would change it to:
- Added `matchesCount` attribute to [[PropertyDataFilterResult]] to optionally tell how many matches there were in a [[PropertyRecord]]. Matches in label and value are separated.
- Added match counting functionality to [[FilteringPropertyDataProvider]]. The returned [[FilteredPropertyData]] now has total matches count and a function get information about a match at specific index.
- Added `highlightedRecordProps` to [[VirtualizedPropertyGridProps]] to allow highlighting specific parts of rendered PropertyRecords.

Each of these items should probably be listed as a separate change in the changefile

Reply:
Updated changeFile and seperated changes